lib/mayaUsd/fileio/shading/shadingModeExporterContext.cpp

```diff
@@ -419,17 +422,25 @@ _UninstancePrim(
     return stage->OverridePrim(path);
 }
 
-UsdPrim
-UsdMayaShadingModeExportContext::MakeStandardMaterialPrim(
-        const AssignmentVector& assignmentsToBind,
-        const std::string& name,
-        SdfPathSet * const boundPrimPaths) const
+UsdPrim UsdMayaShadingModeExportContext::MakeAndBindStandardMaterialPrim(
+    const AssignmentVector& assignmentsToBind,
+    const std::string&      name,
+    SdfPathSet* const       boundPrimPaths) const
+{
+    UsdPrim materialPrim = MakeStandardMaterialPrim(assignmentsToBind, name);
+    BindStandardMaterialPrim(materialPrim, assignmentsToBind, boundPrimPaths);
+    return materialPrim;
+}
```

Comment:
This function is split in two parts since the useRegistry exporter now wants to have the material exported before the binding is done.